bigquery-storage/api/BigQueryStorage.Samples/AppendRowsPending.cs

```diff
@@ -0,0 +1,150 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START bigquerystorage_append_rows_pending]
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Google.Api.Gax.Grpc;
+using Google.Cloud.BigQuery.Storage.V1;
+using Google.Protobuf;
+using static Google.Cloud.BigQuery.Storage.V1.AppendRowsRequest.Types;
+
+public class AppendRowsPendingSample
+{
+    /// <summary>
+    /// This code sample demonstrates how to write records in pending mode
+    /// Create a write stream, write some sample data, and commit the stream to append the rows.
+    /// The CustomerRecord proto used in the sample can be seen in Resources folder and generated C# is placed in Data folder in the Samples directory
+    /// </summary>
+    /// <param name="projectId">The Google cloud project Identifier</param>
+    /// <param name="datasetName">The BigQuery Dataset name</param>
+    /// <param name="tableName">The BigQuery Table name</param>
+    /// <returns>The task</returns>
+    public async Task AppendRowsPendingAsync(string projectId, string datasetName, string tableName)
+    {
+        BigQueryWriteClient bigQueryWriteClient = await BigQueryWriteClient.CreateAsync();
+        // Initialize a write stream for the specified table.
+        // When creating the stream, choose the type. Use the PENDING type to wait
+        // until the stream is committed before it is visible. See:
+        // https://cloud.google.com/bigquery/docs/reference/storage/rpc/google.cloud.bigquery.storage.v1#google.cloud.bigquery.storage.v1.WriteStream.Type
+        WriteStream stream = new WriteStream { Type = WriteStream.Types.Type.Pending };
+        TableName table = TableName.FromProjectDatasetTable(projectId, datasetName, tableName);
+
+        stream = await bigQueryWriteClient.CreateWriteStreamAsync(table, stream);
+
+        // Initialize streaming call, retrieving the stream object
+        BigQueryWriteClient.AppendRowsStream rowAppender = bigQueryWriteClient.AppendRows();
+
+        // Sending requests and retrieving responses can be arbitrarily interleaved
+        // Exact sequence will depend on client/server behavior
+        // Create task to do something with responses from server
+        Task appendResultsHandlerTask = Task.Run(async () =>
+        {
+            AsyncResponseStream<AppendRowsResponse> appendRowResults = rowAppender.GetResponseStream();
+            while (await appendRowResults.MoveNextAsync())
+            {
+                AppendRowsResponse responseItem = appendRowResults.Current;
+                // Do something with responses
+                Console.WriteLine($"Response Append Result is {responseItem.AppendResult}");
```

Comment:
I think this reads better.
```suggestion
                Console.WriteLine($"Appending one row resulted in: {responseItem.AppendResult}");
```